mtags/src/main/scala-3/scala/meta/internal/pc/completions/OverrideCompletions.scala

```diff
@@ -353,18 +358,14 @@ object OverrideCompletions:
     val overrideKeyword: String =
       // if the overriding method is not an abstract member, add `override` keyword
       if !sym.isOneOf(Deferred) || shouldAddOverrideKwd
-      then "override "
+      then "override"
       else ""
 
-    val asciOverrideDef: String =
-      if sym.is(Abstract) then ""
-      else overrideKeyword
-
-    val overrideDef: String = config.overrideDefFormat() match
+    val overrideDefLabel: String = config.overrideDefFormat() match
       case OverrideDefFormat.Unicode =>
-        if sym.is(Abstract) then "🔼 "
+        if sym.is(Deferred) then "🔼 "
```

Comment:
This is irrelevant to the main change, but this change makes `OverrideConfigSuite`'s `unitcode` test closer to Scala2.